apps/dolly-backend/src/main/java/no/nav/dolly/bestilling/service/ImportAvPersonerFraPdlService.java

```diff
@@ -75,7 +75,7 @@ public ImportAvPersonerFraPdlService(TpsfService tpsfService, DollyPersonCache d
     @Async
     public void executeAsync(Bestilling bestilling) {
 
-        log.info("Bestilling med id=#{} er startet ...", bestilling.getId());
+        log.info("Bestilling med id=#{} og type={} er startet ...", bestilling.getId(), getBestillingType(bestilling));
```

Comment:
I stedet for å ha en static metode kan det evt hardkodes direkte da det kun brukes et sted?

Reply:
den også brukes i 6 andre steder, så tenkte at er litt mer lesbare å bruke metode.